decompiler/frontend/binaryninja/frontend.py

```diff
@@ -122,14 +123,18 @@ def from_raw(cls, view: BinaryView):
     def create_task(self, function_identifier: Union[str, Function], options: Options) -> DecompilerTask:
         """Create a task from the given function identifier."""
         function = FunctionObject.get(self._bv, function_identifier)
+        tagging = CompilerIdiomsTagging(self._bv, function.function.start)
+        tagging.run()
```

Comment:
I think that now that compiler idioms handling is integrated into the decompiler and not a separate plugin anymore, the user should probably be able to disable tagging and handling via the config.